### ISSUE TITLE
NGram issue with strings shorter than N

### ISSUE THREAD

**Issue by adamko147** [luozhouyang/python-string-similarity#28]
NGram distance returns wrong value for strings shorter than N.

```python
from strsimpy.ngram import NGram
ng = NGram()
ng.distance("abc", "abc") == 0.0
ng.distance("a", "b") == 0.0 # should be 1.0
```

Distance between `a` and `b` should be `1.0` as the strings are completely different. if N is two, then the code at https://github.com/luozhouyang/python-string-similarity/blob/master/strsimpy/ngram.py#L45 calculates cost of 0 and returns `1.0 * cost / max(sl, tl)`. This returns similarity (which actually is 0, because the strings are completely different). However the code is returning normalized distance, which should be maximum possible here.

This issue seems to be at https://github.com/luozhouyang/python-string-similarity/blob/master/strsimpy/ngram.py#L49 where  it does 
```python
return 1.0 * cost / max(sl, tl)
```
however I think in this case it should return
```python
return 1.0 - cost / max(sl, tl)
```

I'll create PR to address this. 

Thank you

**Comment by adamko147** [comment]
@luozhouyang any chance you can package a publish new version to pypi.org including this fix? Thanks a lot!

**Comment by luozhouyang** [comment]
@adamko147 just released v0.2.1! Thanks for you contribution!
